frb_codegen/src/library/commands/format_rust.rs

```diff
@@ -1,6 +1,6 @@
 use crate::command_run;
 use crate::library::commands::command_runner::{call_shell, check_exit_code};
-use crate::library::commands::format_dart::prepare_paths;
+use crate::library::commands::dart_fix::prepare_paths;
```

Comment:
nit: would be great to change less code in PR, thus maybe we can keep the `prepare_paths` code in the old place (dart format file).

Reply:
Moved but I still changed the name from `format_dart` to `dart_format`.